main.go

```diff
@@ -159,6 +159,10 @@ func forwardRequest(req *http.Request, reqSourceIP string, reqDestionationPort s
 		forwardReq.Host = req.Host
 	}
 
+	if *keepHostHeader {
+		forwardReq.Host = req.Host
+	}
+
```

Comment:
the code on lines 158-161 seems a leftover of the new code on 162-165. should we remove 158-161?

Reply:
Lines 158-161 should be removed as they seem to have been committed incorrectly.